docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -411,7 +411,7 @@ You can also run the test from your IDE directly (be sure you stopped the applic
 
 By default, tests will run on port `8081` so as not to conflict with the running application. We automatically
 configure RestAssured to use this port. If you want to use a different client you should use the `@TestHTTPResource`
-annotation to directly inject the URL of the test into a field on the test class. This field can be of the type
+annotation to directly inject the URL into a field on the test class. This field can be of the type
```

Comment:
Still sounds like something is missing to me  Maybe URL of the tested application into a field ?

Reply:
PR updated, resolving conversation